packages/shorebird_cli/lib/src/commands/preview_command.dart

```diff
@@ -22,7 +23,10 @@ import 'package:shorebird_code_push_client/shorebird_code_push_client.dart';
 /// {@endtemplate}
 class PreviewCommand extends ShorebirdCommand {
   /// {@macro preview_command}
-  PreviewCommand() {
+  PreviewCommand({
+    http.Client? httpClient,
+  }) : _httpClient = httpClient ??
+            retryingHttpClient(LoggingClient(httpClient: http.Client())) {
```

Comment:
might be nice to have a scoped httpClient for this type of thing

Reply:
Agreed, filed https://github.com/shorebirdtech/shorebird/issues/1310